core/remotes/docker/resolver.go

```diff
@@ -285,8 +285,16 @@ func (r *dockerResolver) Resolve(ctx context.Context, ref string) (string, ocisp
 		firstErr         error
 		firstErrPriority int
 	)
+
+	nextHostOrFail := func(i int) string {
+		if i < len(hosts) -1 {
```

Comment:
core/remotes/docker/resolver.go:290: File is not `gofmt`-ed with `-s` (gofmt)

Reply:
Ah, dang! Looks like I had my  "format on save" disabled for some other work 😂 

```diff
--- a/core/remotes/docker/resolver.go
+++ b/core/remotes/docker/resolver.go
@@ -287,7 +287,7 @@ func (r *dockerResolver) Resolve(ctx context.Context, ref string) (string, ocisp
        )

        nextHostOrFail := func(i int) string {
-               if i < len(hosts) -1 {
+               if i < len(hosts)-1 {
                        return "trying next host"
                }
```

Should be fixed now ❤️